docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -34,31 +34,31 @@ Caches can reserve a storage area for their clustered tiers within these server
 Clustered Tier Manager::
 The Ehcache _Clustered Tier Manager_ is the server-side component that gives clustering capabilities to a cache manager.
 Cache managers connect to these to get access to the server's storage resources so that the clustered tiers of caches defined in them can consume those resources.
-Ehcache _Clustered Tier Manager_ at the server side are identified by a unique identifier.
+Each Ehcache _Clustered Tier Manager_ at the server side is identified by a unique identifier.
 Using it, multiple cache managers can connect to the same clustered tier manager to share cache data.
 The clustered tier manager is also responsible for managing the storage of the clustered tier of caches, with the following different options.
 
 Dedicated pool::
-Dedicated pools are a fixed-amount of storage pools allocated to the clustered tiers of caches.
+Dedicated pools are a fixed number of storage pools allocated to the clustered tiers of caches.
 A dedicated amount of storage is allocated directly from server off-heap resources to these pools.
 And this storage space is used exclusively by a given clustered tier.
 
 Shared pool::
-Shared pools are also fixed-amount storage pools, but can be shared by the clustered tiers of multiple caches.
+Shared pools are also a fixed number of storage pools, but can be shared by the clustered tiers of multiple caches.
```

Comment:
:+1: This is OK, providing a bit more detail than the comment from ljacomet above.